src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -288,7 +307,8 @@ private static void fuzzerTestOneInput(
       Throwable cause = e.getCause();
       Class<?> causeClass = cause.getClass();
       // Do not report exceptions declared to be thrown by the method under test.
-      for (Class<?> declaredThrow : throwsDeclarations.get(targetExecutable)) {
+      for (Class<?> declaredThrow :
+          throwsDeclarations.getOrDefault(targetExecutable, new Class[0])) {
```

Comment:
I would prefer explicilty setting `throwsDeclaration` to `new Class[0]` in the initialize functions - we could then also add a comment explaining why this is the right choice for JUnit.

Reply:
Done.